tests/prague/eip663_dupn_swapn_exchange/test_dupn.py

```diff
@@ -0,0 +1,56 @@
+"""
+abstract: Tests [EIP-663: SWAPN, DUPN and EXCHANGE instructions](https://eips.ethereum.org/EIPS/eip-663)
+    Tests for the DUPN instruction.
+"""  # noqa: E501
+
+import pytest
+
+from ethereum_test_tools import Account, Environment, StateTestFiller, TestAddress, Transaction
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from ..eip3540_eof_v1.spec import EOF_FORK_NAME
+from . import REFERENCE_SPEC_GIT_PATH, REFERENCE_SPEC_VERSION
+
+REFERENCE_SPEC_GIT_PATH = REFERENCE_SPEC_GIT_PATH
+REFERENCE_SPEC_VERSION = REFERENCE_SPEC_VERSION
+
+
+@pytest.mark.valid_from(EOF_FORK_NAME)
+def test_dupn_all_valid_immediates(
+    tx: Transaction,
+    state_test: StateTestFiller,
+):
+    """
+    Test case for all valid DUPN immediates.
```

Comment:
This test executes all DUPN instructions in single execution and stores the results in 256 storage slots.

Should I use "parametrize" to create 256 test cases instead?

Reply:
I think this should be fine as is. When we parametrize we try to use edge values, e.g. 0, 256-1, 256, 256+1, but not many in-between.

Reply:
ok then. Anything else to do here?